privacyidea/lib/token.py

```diff
@@ -445,7 +445,8 @@ def get_tokens(tokentype=None, realm=None, assigned=None, user=None,
 def get_tokens_paginate(tokentype=None, realm=None, assigned=None, user=None,
                 serial=None, active=None, resolver=None, rollout_state=None,
                 sortby=Token.serial, sortdir="asc", psize=15,
-                page=1, description=None, userid=None, allowed_realms=None, tokeninfo=None):
+                page=1, description=None, userid=None, allowed_realms=None, tokeninfo=None,
+                excluded_tokeninfo_fields=None):
```

Comment:
please rename to `hidden_tokeninfo_fields` in analogy to `hidden_columns` in audit.py.